views/doc.list.html.twig

```diff
@@ -3,36 +3,61 @@
 {% import _self as macros %}
 
 {% block content %}
-    <div id="searchbar" class="clearfix">
-        <input id="docsearch" type="text" placeholder="Search documentation">
-    </div>
-
-    <h1>Book</h1>
-    <ul>
-    {% for filename, data in book %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>Articles</h1>
-    <ul>
-    {% for filename, data in articles %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-    <h1>FAQs</h1>
-    <ul>
-    {% for filename, data in faqs %}
-        {{ macros.renderFile(filename, data) }}
-    {% endfor %}
-    </ul>
-
-    <h1>API Docs</h1>
-    <p>If you need to work with composer as a library you may also want to check out the <a href="/apidoc/master/index.html">API Docs</a></p>
+    <article role="article">
```

Comment:
you should drop this new element entirely actually. the `content` block defines the full both of the element with `role="main"`. We don't need another wrapping element here (which removes all need to figure out what its ARIA role should be).

Reply:
Done.